e2e-test/e2etest.py

```diff
@@ -0,0 +1,60 @@
+
+from helper import *
+import os
+
+# Setup configuration
+# These values should be injected into the environment before setup
+azure_token = os.environ.get("AZURE_TOKEN")
+bee_name = os.environ.get("BEE_NAME")
+billing_project_name = os.environ.get("BILLING_PROJECT_NAME")
+#TODO: Take in arguments from the command line?

```

Comment:
Agree with this TODO as it'd make it easier to do oneoff running, it'd make sense to do something like:

`commandLineInput || envVariableInput`

Reply:
I think I'm going to save this for future work, when other tests are being added and know what different parameters are needed.